packages/jest-jasmine2/src/__tests__/matchers.test.ts

```diff
@@ -6,8 +6,6 @@
  *
  */
 
-'use strict';
-
 describe('matchers', () => {
```

Comment:
same here - seems like they should be in `expect`

Reply:
This one might be needed?

```js
describe('matchers', () => {
  it('proxies matchers to expect', () => {
    expect(() => expect(1).toBe(2)).toThrowErrorMatchingSnapshot();
  });
});
```

Reply:
Yeah, but it doesn't test anything jasmine? I don't understand what it means by "proxies to expect" - it _uses_ `expect` directly

Reply:
In other words, Jest test suite is integration test that `expect` works in jasmine.

Reply:
Right - if `expect` didn't work in jasmine, the whole test suite would fail. The advantage of using Jest to test Jest 🙂